src/libraries/System.Drawing.Common/src/Interop/Windows/Interop.Comdlg32.cs

```diff
@@ -15,7 +15,7 @@ internal static partial class Comdlg32
         internal static extern bool PrintDlg([In, Out] PRINTDLGX86 lppd);
 
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
```

Comment:
```suggestion
        [StructLayout(LayoutKind.Sequential)]
```
The struct does not contain any marshalable strings anymore